java/java.j2seplatform/arch.xml

```diff
@@ -340,10 +340,55 @@
     </api>
    </li>
   </ul>
+  
+  <api name="API_Ask_attachSourcesQuestion" group="branding" type="export" category="stable">
```

Comment:
Let's agree on following naming policy: `pck.name.KEY_NAME`. I'll update #2685 too.

Reply:
BTW It's a question if the `<api name=""` attribute should not contain the Bundle resource path as well, since the `path/to/Bundle:resourceKey` is the actual API identifier.